public/views/index/body.ejs

```diff
@@ -13,14 +13,14 @@
                             </li>
                             <div class="ui-signin" style="float: right; margin-top: 8px;<% if(signin) { %> display: none;<% } %>">
                                 <% if(allowAnonymous) { %>
-                                <a type="button" href="<%- url %>/new" class="btn btn-sm btn-link"><i class="fa fa-plus"></i> <%= __('New guest note') %></a>
+                                <a type="button" href="<%- url %>/new" class="btn btn-sm btn-success"><i class="fa fa-plus"></i> <%= __('New guest note') %></a>
```

Comment:
I'm not sure if we should use this for not-singed in people, too. As you can see below the sign-in button already uses `btn-success` and from my perspective, there shouldn't be two buttons with this color

Reply:
@SISheogorath The purpose of making these changes is to highlight the `new note` button. Now the button is green, which makes it easier for the user to spot it.

Reply:
@SISheogorath I have changed the color please review.

![screenshot from 2017-10-07 23-21-34](https://user-images.githubusercontent.com/12807846/31310464-547ebebe-abb6-11e7-9546-c58af74c9796.png)